nc-news-react/src/components/Home.jsx

```diff
@@ -1,7 +1,38 @@
+import { useEffect, useState } from "react"
+import { getArticles } from "../axios"
+import { Link } from "react-router-dom"
+
 function Home () {
-    return <div>
-        <h2>Home component</h2>
+  const [articles, setArticles] = useState([])
+  const [loading, setLoading] = useState(true)
+  const [err,setErr] = useState(false)
+
+    useEffect(()=> {
+        getArticles('votes').then((data)=> {
+            setArticles(data)
+            setLoading(false)
+        }).catch((err)=> {
+            setErr(true)
+        })
+    }, [])
+
+    if(loading){return <p>Loading...</p>}
+    console.log(articles)
```

Comment:
Nice to see a loading pattern here 😄 Please don't forget to remove console logs from your code 👀